libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/master-data/dh-market-participant-actor-master-data.component.ts

```diff
@@ -21,39 +21,115 @@ import {
   Input,
   NgModule,
   OnChanges,
+  OnDestroy,
   Output,
 } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { ActorChanges } from '@energinet-datahub/dh/market-participant/data-access-api';
-import { ActorDto } from '@energinet-datahub/dh/shared/domain';
-import { WattFormFieldModule, WattInputModule } from '@energinet-datahub/watt';
-import { TranslocoModule } from '@ngneat/transloco';
+import { ActorDto, ActorStatus } from '@energinet-datahub/dh/shared/domain';
+import {
+  WattDropdownModule,
+  WattDropdownOption,
+  WattFormFieldModule,
+  WattInputModule,
+} from '@energinet-datahub/watt';
+import { TranslocoModule, TranslocoService } from '@ngneat/transloco';
 import { LetModule } from '@rx-angular/template/let';
+import { Subject, takeUntil } from 'rxjs';
 
 @Component({
   selector: 'dh-market-participant-actor-master-data',
   styleUrls: ['./dh-market-participant-actor-master-data.component.scss'],
   templateUrl: './dh-market-participant-actor-master-data.component.html',
 })
-export class DhMarketParticipantActorMasterDataComponent implements OnChanges {
+export class DhMarketParticipantActorMasterDataComponent
+  implements OnChanges, OnDestroy
+{
   @Input() actor: ActorDto | undefined;
   @Output() hasChanges = new EventEmitter<ActorChanges>();
-  changes: ActorChanges = { gln: '', marketRoles: [], meteringPointTypes: [] };
+  changes: ActorChanges = {
+    gln: '',
+    status: 'New',
+    marketRoles: [],
+    meteringPointTypes: [],
+  };
+
+  private destroy$ = new Subject<void>();
+  allStatuses: WattDropdownOption[] = [];
+  statuses: WattDropdownOption[] = [];
+
+  constructor(private translocoService: TranslocoService) {
+    this.translocoService
+      .selectTranslateObject(
+        'marketParticipant.actor.create.masterData.statuses'
+      )
+      .pipe(takeUntil(this.destroy$))
+      .subscribe((statusKeys) => {
+        this.allStatuses = Object.keys(statusKeys)
```

Comment:
suggestion: Use `ActorStatus` to get the Object keys. This way the list of dropdown options will be based on the latest DTOs from the BFF, and not on translations.

```suggestion
        this.allStatuses = Object.keys(ActorStatus)
```